apps/token/src/routes/rewards/home/reward-info.tsx

```diff
@@ -136,7 +121,10 @@ export const RewardTable = ({ reward, delegations }: RewardTableProps) => {
         <KeyValueTableRow>
           {t('received')}
           <span>
-            {format(new Date(reward.receivedAt), DATE_FORMAT_DETAILED)}
+            {format(
+              new Date(Number(reward.receivedAt) / 1000000),
```

Comment:
Nitpick: could you chuck the 1000000 in a var and name it so it's easier to understand what's going on pls?